easybuild/easyconfigs/p/PRINSEQ/PRINSEQ-0.20.4-foss-2018b-Perl-5.28.0.eb

```diff
@@ -0,0 +1,100 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+# Author: Pablo Escobar Lopez
+# sciCORE - University of Basel
+# SIB Swiss Institute of Bioinformatics 
+
+easyblock = 'Tarball'
+
+name = 'PRINSEQ'
+version = '0.20.4'
+versionsuffix = '-Perl-%(perlver)s'
+
+homepage = 'http://prinseq.sourceforge.net'
+description = """A bioinformatics tool to PRe-process and show INformation of SEQuence data."""
+
+toolchain = {'name': 'foss', 'version': '2018b'}
+
+source_urls = ['http://sourceforge.net/projects/prinseq/files/standalone/']
+sources = ['%(namelower)s-lite-%(version)s.tar.gz']
+checksums = ['9b5e0dce3b7f02f09e1cc7e8a2dd77c0b133e5e35529d570ee901f53ebfeb56f']
+
+dependencies = [
+    ('Perl', '5.28.0'),
+]
+
+# these are the perl libraries dependencies
+exts_defaultclass = 'PerlModule'
+exts_filter = ("perldoc -lm %(ext_name)s ", "")
+
+exts_list = [
+    ('ExtUtils::Depends', '0.405', {
+        'source_tmpl': 'ExtUtils-Depends-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/X/XA/XAOC/'],
+        'checksums': ['8ad6401ad7559b03ceda1fe4b191c95f417bdec7c542a984761a4656715a8a2c'],
+    }),
+    ('ExtUtils::PkgConfig', '1.16', {
+        'source_tmpl': 'ExtUtils-PkgConfig-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/X/XA/XAOC/'],
+        'checksums': ['bbeaced995d7d8d10cfc51a3a5a66da41ceb2bc04fedcab50e10e6300e801c6e'],
+    }),
+    ('Pod::Usage', '1.69', {
+        'source_tmpl': 'Pod-Usage-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/M/MA/MAREKR/'],
+        'checksums': ['1a920c067b3c905b72291a76efcdf1935ba5423ab0187b9a5a63cfc930965132'],
+    }),
+    ('Cairo', '1.106', {
+        'source_tmpl': 'Cairo-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/X/XA/XAOC/'],
+        'checksums': ['e64803018bc7cba49e73e258547f5378cc4249797beafec524852140f49c45c4'],
+    }),
+    ('Statistics::PCA', '0.0.1', {
+        'source_tmpl': 'Statistics-PCA-%(version)s.tar.gz',
+        'source_urls': ['https://cpan.metacpan.org/authors/id/D/DS/DSTH/'],
+        'checksums': ['f8adb10b00232123d103a5b49161ad46370f47fe0f752e5462a4dc15f9d46bc4'],
+    }),
+    ('MIME::Base64', '3.15', {
```

Comment:
shouldn't some of these be in the Perl easyconfig directly? They are general purpose?

Reply:
The need never arose, beyond here?

I don't know of anything else that requires this one...